string upload_filepath
---
string result
